js/gitcase.js

```diff
@@ -6,26 +6,94 @@
 
     $.fn.gitcase = function (options) {
 
+        var allRepos = [];
+        var page = 1;
+        var currentRepoNumber = 0;
+        var itemsPerPage = 100;
+
         var defaults = {
-            action: ""
+            action: ''
         };
 
         var opts = jQuery.extend(defaults, options);
 
-        if (opts.action === "request") {
+        if (opts.action === 'request') {
             $(this).click(function () {
-                $.get('github/request.php', {}, function (res) {
+                $.get('github/request.php', {'type': 'authorize'}, function (res) {
                     window.location = res.url;
-                }, "json");
+                }, 'json');
             });
-        } else if (opts.action === "generate") {
+        } else if (opts.action === 'generate') {
             $(this).click(function () {
-                $.get('github/generate.php', {}, function (res) {
-                    console.log(res);
-                }, "json");
-            });
+
+                    $('#generate').addClass('disabled');
+                    $('#octal').css({'display': 'block'});
+                    $('#share').css({'display': 'none'});
+
+                    var finalResult = $("#finalResult");
+                    finalResult.css({'display': 'none'});
+                    finalResult.attr('src', '');
+
+                    allRepos = [];
+                    page = 1;
+                    currentRepoNumber = 0;
+
+                    $.get('github/request.php', {'type': 'user'}, function (res) {
+
+                        $.get(res.url, {'access_token': res.access_token}, function (user) {
+
+                            var totalRepos = parseInt(user.public_repos) + parseInt(user.total_private_repos);
```

Comment:
Line is too long.<br>Identifier 'public_repos' is not in camel case.<br>Identifier 'total_private_repos' is not in camel case.